app/server/skychat/commands/impl/youtube/YoutubePlugin.ts

```diff
@@ -158,7 +158,12 @@ export class YoutubePlugin extends Plugin {
                     connection.send('yt-sync', null);
                 }
                 break;
-        }
+			case 'list':
+				for (let j=0; j<this.queue.length; j++) {
+					this.room.sendMessage(new Message(this.queue[j].video.title + ', added by ' + this.queue[j].user.username, UserController.getNeutralUser()));
+				}
```

Comment:
There are a few things that I think could be better here:
- Only one message should be sent instead of one / item in the queue
- The message should only be sent to the user that asked the list. By using this.room.sendMessage, the message is broadcasted to everyone in the room.
Here is a fix:
```typescript
const message = new Message('List:', UserController.getNeutralUser());
for (let j=0; j < this.queue.length; j++) {
    message.append(this.queue[j].video.title + ', added by ' + this.queue[j].user.username);
}
connection.send("message", message);
```

Reply:
yes it isn't